conda_recipes/aftereffects-saber/recipe/build_win.sh

```diff
@@ -1,9 +1,8 @@
 #!/bin/sh
 set -xeuo pipefail
 
-mkdir -p $PREFIX/opt
+AE_LOCATION="$PREFIX/aftereffects"
```

Comment:
You should be able to remove this from the recipe, and instead use the `aftereffects` plugin as a dependency. The variable `AE_LOCATION` will then get defined when the host environment is activated.